.changes/unreleased/tinymce-TINY-11273-2024-09-24.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Autocompleter would not activate after font size change.
```

I am wondering if this should be more generic
```suggestion
body: Autocompleter would not activate after applying an inline format like font size in some cases.
```